packages/cli-kit/bin/fetch-notifications.js

```diff
@@ -0,0 +1,21 @@
+import fetch from 'node-fetch'
+import {notificationsUrl} from '../dist/public/node/notifications-system.js'
+import {cacheStore} from '../dist/private/node/conf-store.js'
```

Comment:
I don't think you can do this. This paths won't exist once we bundle the CLI.
You can try by doing `pnpm bundle` and then running the CLI

Reply:
Ah, good point. I forgot to try with the snapshot, let me check it...